RT_SYCL/camera.hpp

```diff
@@ -86,6 +86,9 @@ class camera {
     lens_radius = aperture/2;
   }
 
+  /* Computes ray from camera passing through 
```

Comment:
Use Doxygen style for it starting with `/**` so this comment can be captured as some documentation to generate in the future some pages like in https://trisycl.github.io/triSYCL/Doxygen/triSYCL/html/index.html

Look for example at typical triSYCL code for inspiration.
Tweak also the indent of the comment.

For more specific syntax: https://www.doxygen.nl/manual/docblocks.html